lib/engine.js

```diff
@@ -52,19 +53,34 @@ function nextTick() {
   towers.forEach(tower => tower.shoot(game.retrieveAliveEnemiesOnBoard()));
 }
 
-requestAnimationFrame(function gameLoop() {
+requestAnimationFrame(gameLoop);
+
+function gameLoop() {
   ctx.clearRect(0, 0, canvas.width, canvas.height);
 
   drawBoard(ctx);
   drawEnemies(ctx);
   drawTowers(ctx);
   nextTick();
-  game.checkStatus();
-  // console.log(game.enemies[0].health);
+  if (game.checkStatus() === 'ongoing'){
```

Comment:
I really like this implementation of breaking out of the current 'ongoing' gameLoop and rendering a different animation (albeit a blank one right now).